### DISCUSSION TITLE
System.DllNotFoundException: 'Magick.Native-Q8-arm64.dll assembly:<unknown assembly> type:<unknown type> member:(null)'

### DISCUSSION THREAD

**Discussion by Grobix34** [dlemstra/Magick.NET#1137] [Help]
Hello, I´m getting this error when i try to do something with this nuget package. I can see that is a library error but i just cant solve it by myself. Im using Xamarin.Forms to do the application.
MagickNET.Initialize(); (The error jumps here)
            using (var image = new MagickImage("filename.png")) 
            {
                image.Write("FILENAME.PCX");
            }

**Comment by dlemstra** [comment]
What is the operating system that you are trying to use this library on?

**Reply by Grobix34** [reply]
Wow, that´s fast. Windows10x64

**Reply by Grobix34** [reply]
I´m using the Magick.NET-Q8-AnyCPU btw. Forgot to say that

**Reply by dlemstra** [reply]
Are you using the latest version? It don't think that this library is compatible with Xamarin.Forms. Are you creating a Windows App?

**Reply by Grobix34** [reply]
Yes, i´m using the latest version and no, it´s and android app. I need to convert a photo to pcx format for a thermal printer.

**Reply by dlemstra** [reply]
It looks like the arm64 version of the library is used on android. I think I will need to make some patches to prevent this from happening. Could you switch to the `-x64` version and give it another try?

**Reply by Grobix34** [reply]
Keeps getting the error with both of them, the arm64 and -x64

**Reply by dlemstra** [reply]
I might be possible that you will need to copy a native library to the bin folder but I have no experience with Xamarin.Forms. There should be a `Native` `.so` file in the `bin` folder.

**Reply by Grobix34** [reply]
![image](https://user-images.githubusercontent.com/99818036/154297589-84f3e570-a118-4d1a-b1db-74591433aade.png)
Over here??

**Reply by Grobix34** [reply]
I mean. Copy the dll file from the nuget package an paste it in there????

**Reply by dlemstra** [reply]
There is a `.so` file in one of the `native` folder of the `NuGet` package. I also don't see the `Magick.NET` binaries in that folder?

**Reply by Grobix34** [reply]
I can´t see the .so file i any of the folders

**Reply by Grobix34** [reply]
I can se a dll file in the native folder but no so
![image](https://user-images.githubusercontent.com/99818036/154299454-35650751-4114-4c47-ae24-35710a7dd534.png)

**Reply by Hackmodford** [reply]
I'm having the same issue on macOS. Trying to use it in a mac catalyst app.
